config/routes.rb

```diff
@@ -1,5 +1,23 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: 'pages#home'
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get 'about', to: 'pages#about', as: :about
+  get 'dashboard', to: 'pages#dashboard', as: :dashboard
+  get 'search', to: 'pages#search', as: :search
+
+  resources :developers
+
+  resources :companies do
+    resources :jobs, only: [:new, :create]
+  end
+
+  resources :jobs, only: [:show, :index, :edit, :update, :destroy] do
+    resources :applications, only: [:new, :create]
+  end
+
+  resources :applications, only: [:show, :edit, :update, :destroy]
```

Comment:
For the MVC, we probably don't need to add the ability to save and edit applications, do we? No harm to leave the route in I suppose, but we should agree whether we need this functionality.

Reply:
saving an application would be important, don't you think? do you mean edit an application? edit application could also be just to accept it(set the status column to accepted?)

Reply:
We can do this, just need to be careful that an application cannot be edit once sent to a company. For, now let's not worry to much, no harm in have the route there.